src/boot_config/winrm_bootstrap.txt

```diff
@@ -0,0 +1,47 @@
+<powershell>
```

Comment:
did you come up with this file on your own or did you copy it from somewhere?  if copied it from somewhere could you please cite the source?

Reply:
I copied this from https://github.com/Sage-Bionetworks-IT/packer-base-winserver2019/blob/v0.0.1/src/boot_config/winrm_bootstrap.txt and added a comment with that link

Reply:
ahh, actually you should probably cite from here https://developer.hashicorp.com/packer/integrations/hashicorp/amazon/latest/components/builder/ebs#connecting-to-windows-instances-using-winrm

Reply:
oh, awesome, thanks for the better link